### ISSUE TITLE
Text labels of two circles can overlap

### ISSUE THREAD

**Issue by scottmartinnet** [benfred/venn.js#18]
I've encountered a situation where two pieces of text can overlap each other when their circles get drawn in a similar position.

![venn](https://cloud.githubusercontent.com/assets/1265917/2542839/23b66dea-b5ec-11e3-8c2d-40944a17daca.png)

```
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script><script src="d3.js"></script><script src="venn.js"></script><script type="text/javascript">$(document).ready(function() {    var sets = [{"id":40,"label":"HBO","size":1792283,"radius":167.2,"y":187.2,"x":187.2},{"id":42,"label":"Game of Thrones","size":1210069,"radius":137.3845982071701,"y":187.2,"x":225.56428991864962}];    var overlaps = [{"id":40,"sets":[1,0],"size":1186304}];    var combined = venn.venn(sets, overlaps);    var diagram = venn.drawD3Diagram(d3.select('#overlap-chart'), combined, $('#overlap-chart').width(), 375);});</script><div id="overlap-chart" style="width: 624px; height: 374px;"></div> 
```

**Comment by benfred** [comment]
after last commit (3d281c9) output on your data looks like:

![screen shot 2015-05-12 at 11 53 37 pm](https://cloud.githubusercontent.com/assets/69536/7605250/3637bdc0-f902-11e4-991e-5b1759a1023f.png)
